pages/index.js

```diff
@@ -0,0 +1,78 @@
+import Head from 'next/head';
+import Header from '../components/Header.js';
+import Footer from '../components/Footer.js';
+import React from 'react';
+
+export default function Home() {
+  return (
+    <div className="container">
+      <Head>
+        <title>Bist du Teppich </title>
+        <link rel="icon" href="/favicon.png" />
+        <link
+          href="https://fonts.googleapis.com/css2?family=DM+Mono&display=swap"
```

Comment:
You can do global things like this that you want to load on every page in a special, separate component (very often called `<Layout />`)